products-service/functions/createProduct/createProduct.js

```diff
@@ -0,0 +1,103 @@
+import { Client } from "pg";
+
+const handleResponse = (products = {}, status = 200) => ({
+  headers: {
+    "Content-Type": "application/json",
+    "Access-Control-Allow-Methods": "*",
+    "Access-Control-Allow-Origin": "*",
+  },
+  statusCode: status,
+  body: JSON.stringify(products),
+});
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+
+const credentials = {
+  user: PG_USERNAME,
+  host: PG_HOST,
+  database: PG_DATABASE,
+  password: PG_PASSWORD,
+  port: PG_PORT,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+  connectionTimeoutMillis: 5000,
+};
+
+let data_export = {},
+  error_code = 200;
+
+export const handler = async (event) => {
+  console.log(event);
+
+  const client = new Client(credentials);
```

Comment:
It would be better to create a separate module (product service) that would handle all database connections and queries